api/clients/v2/metrics/accountant.go

```diff
@@ -69,6 +81,9 @@ var NoopAccountantMetrics AccountantMetricer = new(noopAccountantMetricer)
 func (n *noopAccountantMetricer) RecordCumulativePayment(_ string, _ *big.Int) {
 }
 
+func (n *noopAccountantMetricer) RecordReservationPayment(_ float64) {
+}
+
```

Comment:
I don't mind if you leave this alone since there is pre-existing code. I usually don't create mock metrics instances, I just make it so that metrics functions become no-ops if `this` is `nil`.

Reply:
Makes sense. Let's leave as-is for now, to avoid changing pre-existing code